params/config.go

```diff
@@ -111,6 +113,8 @@ var (
 		Istanbul: &IstanbulConfig{
 			Epoch:          17280,
 			ProposerPolicy: 2,
+			BlockPeriod:    5,
+			RequestTimeout: 10000,
```

Comment:
I'm not sure about setting the timeout for Baklava to 10000, given what we discovered when we looked into how the configuration works.  It was 10000 initially due to the value in the genesis json, but when validators starting using `--baklava`, it overwrote it in the stored configuration with the default which is 3000.  I verified this today as follows:
1. add logging all over the place to see exactly how the timeout parameter is passed around and what it is in the end
1. run `geth init` using the baklava genesis
2. with the same datadir, run geth without `--baklava`, verify the timeout it uses is 10000
3. with the same datadir, run geth with `--baklava`, verify the timeout is uses is now 3000

At this point, because of Churrito, we know everyone is using `--baklava`, so everyone has 3000.  With this change, it will overwrite it back to 10000.  Is that explicitly the intention, to go back to 10000 to agree with the genesis json?  If not, I would prefer to use 3000 here.

Reply:
I'm convinced!. If everyone is running with --baklava, then let's use `3000` here, and modify the genesis.json file just in case... (or not... i need to check if the number is part of the genesis block hash)